responder/models.py

```diff
@@ -107,8 +107,8 @@ def __init__(self, scope, receive, api=None):
     @property
     def session(self):
         """The session data, in dict form, from the Request."""
-        if "Responder-Session" in self.cookies:
-            data = self.cookies["Responder-Session"]
+        if self.api.session_cookie in self.cookies:
```

Comment:
`self.api` can be None as per the constructor arguments. In that case this will fail.

Reply:
Fixed that